src/main/java/edu/kis/powp/jobs2d/command/manager/LoaderCommand.java

```diff
@@ -0,0 +1,5 @@
+package edu.kis.powp.jobs2d.command.manager;
+
+public interface LoaderCommand {
+    public LoadedCommand loadCommandFromExternalSource();
```

Comment:
This operation should accept String parameter to be useful (as an interface).

Reply:
Interace is now accepting String parameter.